spec/controllers/account/companies_controller_spec.rb

```diff
@@ -0,0 +1,143 @@
+require 'rails_helper'
+
+RSpec.describe Account::CompaniesController, type: :controller do
+  let!(:company) { create(:company) }
+  before(:each) { sign_in User.last }
```

Comment:
why User.last? This is wrong. Use owner of a company.

Reply:
and remember ... in many cases when you have code like ".first", ".last" there is a bug

Reply:
My bad, company.user should be a better choice